basics/a44.go

```diff
@@ -0,0 +1,16 @@
+//multiple constant declarations
+
+package main
+
+import "fmt"
+
+func main() {
+	const (
+		base_frequency = 2.40
+		turbo_frequency = 5.30
```

Comment:
Use camelcase instead of snake case. At least that is what GoLand is complaining, not sure if that is the standard with GoLand. Do you know?

Reply:
okay. switching to camelCase.